src/include/daos/pool_map.h

```diff
@@ -129,13 +133,13 @@ pool_target_id_list_free(struct pool_target_id_list *id_list);
  */
 struct pool_buf {
 	/** checksum of components */
-	uint32_t		pb_csum;
+	uint32_t	pb_csum;
 	/** summary of domain_nr, node_nr, target_nr, buffer size */
-	uint32_t		pb_nr;
-	uint32_t		pb_domain_nr;
-	uint32_t		pb_node_nr;
-	uint32_t		pb_target_nr;
-	uint32_t		pb_padding;
```

Comment:
Actually, don't we need to keep the padding here for 64 bit alignment?

Reply:
Good point.